### DISCUSSION TITLE
I made this flower/jellyfish!

### DISCUSSION THREAD

**Discussion by jdavis-codes** [inca/blender-differential-growth#8] [Show and tell]
this tool is incredible, thank you :)

[Reddit](https://www.reddit.com/r/blender/comments/tlytmp/flower_or_jellyfish/?utm_source=share&utm_medium=web2x&context=3)

https://user-images.githubusercontent.com/79899854/159851116-fe781219-4002-4c09-b480-64dccaa8358f.mp4

**Comment by inca** [comment]
Amazing work! ✨  Thanks so much for sharing this 💖
